### PR TITLE
MAINT: pysat 3.2.0 RC compliance

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses # (issue)

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.  Please see ``CONTRIBUTING.md`` for more guidelines.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality
      to not work as expected)
- This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details for
your test configuration

- Test A
- Test B

**Test Configuration**:
* Operating system: Hal
* Version number: Python 3.X
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

**Comment by aburrell** [comment]
Wondering if the additional download requirements are causing the shut off?

**Comment by jklenzing** [comment]
> Wondering if the additional download requirements are causing the shut off with the pysat RC?

It's failing with the previous release as well

**Comment by jklenzing** [comment]
Reworked my local config. Passes with pysat 3.1.0.

The failing test is `test_load_empty` for cosmic gps, tag='gfsprf'

We could add test_load_empty to the tests that are currently skipped

**Comment by aburrell** [comment]
> We could add test_load_empty to the tests that are currently skipped

That's a good idea, since this a test that enforces a standard we weren't strict on before.

**Comment by jklenzing** [comment]
> > We could add test_load_empty to the tests that are currently skipped
> 
> That's a good idea, since this a test that enforces a standard we weren't strict on before.

Found more issues, including with unchanged tests. I think this might be related to the imports / memory

**Review by JonathonMSmith — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         test_config: ["latest"]
         include:
           - python-version: "3.9"
-            numpy_ver: "1.21"
+            numpy_ver: "1.22"
```
Comment (jklenzing) [inline .github/workflows/main.yml:20]:
```suggestion
            numpy_ver: "1.23"
```

**Comment by jklenzing** [comment]
Merging this in to get develop up to date. Documented issues in operational workflow in #54.

**Pull request merged**